zhaquirks/xiaomi/aqara/tvoc.py

```diff
@@ -66,6 +67,24 @@ async def bind(self):
         return result
 
 
+class TVOCDisplayUnit(t.enum_factory(t.uint8_t)):
```

Comment:
Using `t.enum_factory(t.uint8_t)` is fine, but directly using `t.enum8` is preferable.

Reply:
Does not work. The device reject it. Enum8 maps to some standard type identifier.

Reply:
It should not. Are you sure you used the zigpy `enum8`?: https://github.com/zigpy/zigpy/blob/a3e449582fe2a1203316dce0955ccfe19d6fdc3b/zigpy/types/basic.py#L461

Reply:
Ah, I see. That’s probably why most custom attributes for Xiaomi devices just have their type as `t.uint8_t` in zha-quirks and then have an enum for the config entity in ZHA (as that does not change the "Zigbee type").